frontend/components/adopter/ViewPets/data/filters.ts

```diff
@@ -0,0 +1,196 @@
+import { Animal } from "@contract";
+import dayjs from "dayjs";
+import customParseFormat from "dayjs/plugin/customParseFormat";
+import utc from "dayjs/plugin/utc";
+
+dayjs.extend(customParseFormat);
+dayjs.extend(utc);
+
+export const SpeciesFilterOptions = [
+    {
+        options: [
+            {
+                label: "Cat",
+                value: Animal.Species.Cat,
+            },
+            {
+                label: "Dog",
+                value: Animal.Species.Dog,
+            },
+            {
+                label: "Others",
+                value: Animal.Species.Others,
+            },
+        ],
+    },
+];
+
+export enum AgeFilter {
+    Above0Months,
+    Above6Months,
+    Above12Months,
+    Above36Months,
+    Above72Months,
+    Above120Months,
+}
+
+export const AgeFilterOptions = [
+    {
+        title: "Under a year",
+        options: [
+            {
+                label: "1-6 months",
+                value: AgeFilter.Above0Months,
+            },
+            {
+                label: "7-12 months",
+                value: AgeFilter.Above6Months,
+            },
+        ],
+    },
+    {
+        title: "Beyond a year",
+        options: [
+            {
+                label: "1-3 years",
+                value: AgeFilter.Above12Months,
+            },
+            {
+                label: "4-6 years",
+                value: AgeFilter.Above36Months,
```

Comment:
Might be good to rename and/or add unit tests on these filters as the labels do not match the enum name `Above36Months` and the filter does not match the label too. The current value would include 3+ years old which is inaccurate to 4-6 years.